pkg/gatewayserver/io/io.go

```diff
@@ -407,10 +407,17 @@ func (c *Connection) ScheduleDown(path *ttnpb.DownlinkPath, msg *ttnpb.DownlinkM
 			break
 		}
 	}
-	phy, err := band.GetByID(fp.BandID)
+
+	phyFromFP, err := band.GetByID(fp.BandID)
+	if err != nil {
+		return false, false, 0, err
+	}
+
+	phy, err := phyFromFP.Version(request.GetLorawanPhyVersion())
```

Comment:
I think this should fall back to `1.1-a` on `nil`, for backwards compatibility reasons.

Reply:
I considered it. But I think that it's better to fail downlinks. If we just fallback and allow them, it'll be difficult to debug when customers come back with mismatched downlink issues.

Reply:
We should stay compatible with the old behavior, if it were only for Packet Broker and LoRaWAN Backend Interfaces support. This field can be optional, and if it isn't set, do not version the PHY and assume the latest version is used.

Reply:
Ok sounds good. Changes coming up.